docs/_templates/versions.html

```diff
@@ -14,16 +14,6 @@
       {% endfor %}
     </dl>
     {% endif %}
-    <!-- Uncomment the following and for demo purposes. -->
```

Comment:
This doesn't seem necessary - without this code, we still get 8 languages showing up in the sample build. I believe that is from our Python extension.